app/build.gradle

```diff
@@ -20,8 +20,8 @@ dependencies {
     // Utils
     implementation 'in.yuvi:http.fluent:1.3'
     implementation 'com.google.code.gson:gson:2.8.5'
-    implementation ("com.squareup.okhttp3:okhttp:$OKHTTP_VERSION"){
-        force = true //API 19 support
+    implementation ("com.squareup.okhttp3:okhttp:$OKHTTP_VERSION!!"){
+        //force = true //API 19 support
```

Comment:
Forcing dependency versions using force = true on a first-level dependency has been deprecated.

Refer to [Gradle documentation](https://docs.gradle.org/7.5/userguide/upgrading_version_5.html#forced_dependencies).

Reply:
> Also, the minSdkVersion is 29 for this project.

We've pushed it back to 21, recently. I hope that doesn't affect this change 🙂